core/src/main/java/org/apache/calcite/sql2rel/StandardConvertletTable.java

```diff
@@ -2135,11 +2137,21 @@ private static class TimestampAddConvertlet implements SqlRexConvertlet {
       final RexNode op2;
       switch (call.operandCount()) {
       case 2:
-        // BigQuery-style 'TIMESTAMP_ADD(timestamp, interval)'
-        final SqlBasicCall operandCall = call.operand(1);
-        qualifier  = operandCall.operand(1);
-        op1 = cx.convertExpression(operandCall.operand(0));
-        op2 = cx.convertExpression(call.operand(0));
+        if (call.getOperator() == SqlLibraryOperators.ADD_MONTHS) {
+          qualifier = new SqlIntervalQualifier(TimeUnit.MONTH, null, SqlParserPos.ZERO);
```

Comment:
Add Comment like _BigQuery-style 'TIMESTAMP_ADD(timestamp, interval)_ here , would be easier to understand. WDYT

Reply:
done